### DISCUSSION TITLE
Website discussion: iips-acm.github.io

### DISCUSSION THREAD

**Discussion by adeen-s** [iips-acm/iips-acm.github.io#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. This discussion page is meant to discuss the details about the website for our college ACM Chapter.
  
  * The website will serve as the digital face of our community.
  * It will list our previous and upcoming events.
  * The website will host important public documents such as our community's Code of Conduct and Community Guidelines.
  * Remember that this is a community effort and we have to build this website together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. 
  We are looking to gather ideas about the look and feel of the website, and the technology stack to be used for building the website.

Looking forward to hearing your ideas.

**Comment by oswalgopal** [comment]
# Hello 
I am Gopal Oswal 3rd Year Student. I am a full-stack developer with expertise in developing web and mobile apps.
I have good leadership skills and have community experience with GDG and GCDC Indore.

- For website development, I will suggest using simple HTML CSS and js as I think that most of our members know about these domains. 
- For the backend perspective I will suggest using firebase since it is free up to some extends so we can save and add our events from it.
- We can also use GitHub issues for raising bugs for our website.

Thanks 😊

**Comment by tarunchoudhary1805** [comment]
**Hello **
 I am Tarun Choudhary 2nd year MCA student . I am a full stack MERN developer having good knowledge of Reactjs , Redux and Redux toolkit . 
- we will make our website in MERN so that if we want to make it PWA or a mobile application we will do it . 
- using react we dont have to write too much code in html . 
- as gopal sir mentioned we can use github issues as well as we can use github for tasks management . 
- we can also use Bootstrap so we dont have to write too much css .
